src/components/AddItem.js

```diff
@@ -46,6 +67,9 @@ const AddItem = ({ token }) => {
             value={formData.itemName}
             onChange={handleFormChange}
           />
+          {hidden ? (
```

Comment:
This is an unintuitive name to me. When I read this it says:

when hidden is true, then show this message. Maybe `hasDuplicate` and `setHasDuplicate`? something like that. That way it would be more like:

If this thing has a duplicate, then show this error message.